script/testdata/README.md

```diff
@@ -4,7 +4,7 @@
 
   - Create a directory named `deps`.
 
-  - Clone https://github.com/nervosnetwork/ckb-c-stdlib into `deps`.
+  - Clone https://github.com/XuJiandong/ckb-c-stdlib.git with branch `syscall-spawn` into `deps`.
```

Comment:
we may need to add a FIXME to remember to clean this up if we don't want to resolve it right now.